### ISSUE TITLE
UI: Make a react carousel

### ISSUE THREAD

**Issue by siwamsingh** [dscnsec/Stake-frontend-clone#11]
## Description
- make a component for react carousel

## Task
- install carousel using the `npm i react-multi-carousel `.
- Initialize Carousel.jsx in `src/components/Carousel` directory ( ⚠️ NOTE: If the components and Carousel folders haven't been created yet, please go ahead and create them..)
- use react-multi-carousel  to make Carousel.jsx componenet.
- for logos use any logo you find. ( Note: all the logos should have the same kind of style like in image shown below)
- visit ofiicial stake website for more info.

## Implementation 🔽
![WhatsApp Image 2024-10-22 at 12 51 36 PM](https://github.com/user-attachments/assets/f1c89d34-6964-41bb-810c-9933b951ef62)

**Comment by mithelan** [comment]
@siwamsingh Can i take this up?

**Comment by siwamsingh** [comment]
> @siwamsingh Can i take this up?

Yes you can. Just make sure to follow the insttruction. Happy Coding

**Comment by dakshsinghrathore** [comment]
@mithelan any updates?
